optimizely.php

```diff
@@ -232,6 +232,24 @@ public function get_experiment_results( $experiment_id, $options = array() ) {
 	}// end get_experiment_results
 
 	/**
+	 * Get the top-level results (stats) of an experiment, including number of visitors,
+	 * number of conversions, and chance to beat baseline for each variation.
+	 */
+	public function get_experiment_stats( $experiment_id, $options = array() ) {
+		// @TODO: support options
+		// @TODO: check for 503 in case this endpoint is overloaded (from docs)
+		$extra = '';
+		if ( $options ) {
+			$extra = '?' . http_build_query( $options );
+		}//end if
+
+		return $this->request( array(
+			'function' => 'experiments/' . abs( intval( $experiment_id ) ) . '/stats' . $extra,
+			'method' => 'GET',
+		) );
+	}// end get_experiment_results
```

Comment:
Hey @JWMarchant can I get you to update that function end comment to show the correct method name. Otherwise this looks good to me and I'll merge once updated.